### ISSUE TITLE
How can I provide vertical printing shortcut for text element and pdf support for preview/print ? 

### ISSUE THREAD

**Issue by burakgokcinarr** [salgum1114/react-design-editor#257]
I need a design tool and I'm looking into the project. I don't have much experience with Fabric.js. What I need for this design tool are elements like Text, Barcode. When I drag a Text element to the canvas area, it prints horizontally by default. My goal is how can I add a property to print this TextBox element vertically? (Or it can be a rotation button connected to the part that changes the orientation of the element on the canvas).

In addition, the preview only makes it as an Image, I convert it to a pdf in the canvas area and send the “print” command to the printer and get output.

In short, what should I do for text element vertical printing, pdf preview and print features or is there a plan for this?

Translated with DeepL.com (free version)
